Documentation/guides/tracing.md

```diff
@@ -319,6 +326,13 @@ dotnet build foo.csproj
 dotnet trace collect --format speedscope -- dotnet build "-t:Run" -bl --no-restore foo.csproj
 ```
 
+On MacOS (and possibly linux) you need to pass `-p:UseSharedCompilation=false` otherwise the trace will hang.
+
+```dotnetcli
+dotnet restore foo.csproj
+dotnet trace collect --format speedscope -- dotnet build "-t:Run" -bl --no-restore foo.csproj -p:UseSharedCompilation=false
```

Comment:
If the `-p:UseSharedCompilation=false` works, do we not need the quotes around `"-t:Run"` anymore? Maybe that was a bug fixed at some point?

Reply:
The quotes for `-t:Run` never should have been needed.  You'd need quotes if you wanted to specify *multiple* `;`-separated targets, e.g. `"-t:Restore;Run"`.

Now, it might be handy to have quotes *anyway* to make editing the command-line easier…. but as-is, the quotes here are not needed, and never were.

Reply:
…but then there's the subsequent paragraph, doh:

>  found that `"` is necessary when `:` characters are present in the command. This appears to be some kind of argument parsing issue with `dotnet trace`.

So maybe the double-quotes *are* needed.